src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -366,51 +363,12 @@ public function askQuestion(Question $question)
 
         if ($this->input->isInteractive()) {
             $this->newLine();
-            $this->bufferedOutput->write("\n");
+            $this->newBufferedLine();
```

Comment:
no need for a new method, we already have the `writeBuffer` method

Reply:
I think it makes sense to encapsulate a new line case (used in several cases in the new class) because it hides the real used symbol. Somewhere PHP_EOL used, somewhere "\n" and encapsulated method helps to not worry about it.